src/plugins/visualizations/public/embeddable/visualize_embeddable.ts

```diff
@@ -391,6 +391,13 @@ export class VisualizeEmbeddable
       syncColors: this.input.syncColors,
       uiState: this.vis.uiState,
       inspectorAdapters: this.inspectorAdapters,
+      executionContext: this.deps.start().core.executionContext.create({
+        type: this.vis.params.type,
+        name: this.vis.type.name,
```

Comment:
Both of these values are the same in practice, is that intentional? Seems like if they are going to be the same value they should use the same accessor.

Reply:
Name is a public name of `type`, because sometimes we report such discrepancy: `"type":"gauge","name":"tagcloud"`

Reply:
I was going to use `type: visualiazation`, but now I'm not sure it's useful. I will use `name` as `type` and remove `name` field